packages/components/addon/components/hds/side-nav/section/item.hbs

```diff
@@ -0,0 +1,7 @@
+<li class="hds-side-nav__list-item">
+  <span class="hds-side-nav__item" ...attributes>
+    <span class="hds-side-nav-label hds-typography-body-200">
+      {{yield}}
+    </span>
+  </span>
+</li>
```

Comment:
Thanks, yeah I need to look at this one more. Should consumers have to use `SideNav::List::Item` or could it just be an internal component? 

(I was trying to avoid consumers needing to use a separate component to contain the Link, etc. content themselves...)

Reply:
I refactored this so that the "Item" component is now the same as the HTML list-item (li) element. It must now be used to wrap Link components.

I still need to work on the "BackLink" component after this.